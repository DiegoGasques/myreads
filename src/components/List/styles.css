.list {
  list-style-type: none;
  padding: 0;
  margin: 0;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-wrap: wrap;
}
