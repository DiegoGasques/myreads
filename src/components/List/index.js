import React from "react";
import PropTypes from "prop-types";
import classNames from "classnames";

import "./styles.css";

function List({ className, children }) {
  return <ul className={classNames("list", className)}>{children}</ul>;
}

List.propTypes = {
  className: PropTypes.string,
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ])
};

export default List;
