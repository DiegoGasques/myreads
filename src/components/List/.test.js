import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import List from "./index";

it("Should match the component snapshot", () => {
  const wrapper = shallow(
    <List className="row">
      <li>Child</li>
      <li>Child</li>
    </List>
  );
  expect(toJSON(wrapper)).toMatchSnapshot();
});
