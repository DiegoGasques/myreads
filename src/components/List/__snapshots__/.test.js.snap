// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match the component snapshot 1`] = `
<ul
  className="list row"
>
  <li>
    Child
  </li>
  <li>
    Child
  </li>
</ul>
`;
